crates/lsp/src/rust_analyzer/to_proto.rs

```diff
@@ -1,60 +0,0 @@
-// --- source
-// authors = ["rust-analyzer team"]
-// license = "MIT OR Apache-2.0"
-// origin = "https://github.com/rust-lang/rust-analyzer/blob/master/crates/rust-analyzer/src/lsp/to_proto.rs"
-// ---
-
-//! Conversion of rust-analyzer specific types to lsp_types equivalents.
-
-use super::{
-    line_index::LineIndex,
-    text_edit::{Indel, TextEdit},
-};
-use settings::LineEnding;
-use tower_lsp::lsp_types;
-
-pub(crate) fn text_edit(
```

Comment:
These did not get removed, they got lifted up to the top level `to_proto.rs`, and are now implemented on top of our own `to_proto::range()` helper, not the one from biome_lsp_converters